injected/integration-test/message-bridge-apple.spec.js

```diff
@@ -49,7 +49,7 @@ test('message bridge when enabled (apple)', async ({ page }, testInfo) => {
     await isolated.simulateSubscriptionMessage('exampleFeature', 'onUpdate', { abc: 'def' });
 
     // get all results
-    const results = await pageWorld.runTests();
+    const results = await pageWorld.results();
```

Comment:
I renamed from `runTests` to just `results` since it's more accurate. The collector doesn't manually start any test runs (unless you choose to, with a button click etc)